packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -81,16 +82,19 @@ module('Integration | operator-mode', function (hooks) {
   );
   setupServerSentEvents(hooks);
   setupMatrixServiceMock(hooks);
+  setupWindowMock(hooks);
   let noop = () => {};
 
   hooks.afterEach(async function () {
     localStorage.removeItem('recent-cards');
+    localStorage.removeItem('recent-files');
```

Comment:
I think this clears “real” `localStorage`, if you want to use the mocked one in tests you need `import window from 'ember-window-mock'` and then `window.localStorage`

BUT I think none of this is needed anymore, if everything is set up to use the conditional `window` it’s cleaned up automatically and these `removeItem`s are redundant.